connector/mqtt/vernemq/broker/bin/vernemq.sh

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env bash
+
```

Comment:
Did you copy this file from some place?

Reply:
I asked because of license, but maybe It shouldn't be a problem

Reply:
Yes I did
I just copy this repo https://github.com/sportebois/docker-vernemq/

Reply:
The repo license is apache 2,0, what is good.
I think you just need refer the repo in the file
What do you think @rascaraficci ?

Reply:
As this file was licensed under Apache 2.0 by the author, I think there is no problem to copy it. My suggestion is to add a NOTICE file saying that we copy/use it and the original one is available at ...